suite-native/alerts/package.json

```diff
@@ -14,6 +14,7 @@
         "@suite-common/icons-deprecated": "workspace:*",
```

Comment:
It seems that send btn always visible in selecting amount to send is not resolved. Not sure if it should be included, just mentioning this.